### ISSUE TITLE
All: User Account name is highlighted even when unselected

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#6]
**URL**: https://hcdev.openplus.ca/en
**User Agent**: Chrome/Firefox
**Language**: both

**Issue Description:** In the menu on top, the user account name is highlighted even when user is on another page. This can confuse the user of where they are at in the menu. 

**Steps to reproduce:**  Visual

**Expected result**: The correct location should be highlighted so the user knows where they are at. 
![image](https://user-images.githubusercontent.com/65728700/126989274-bc2613a0-a487-4a57-86b0-884b66cae0aa.png)

**Comment by davidgotlieb** [comment]
This is behaving as expected. 

This button controls the secondary menu, and does not redirect to any pages. It is only highlighted when the secondary menu is active (as seen in your screenshot), and is black when the secondary menu (Help, Log out, Edit profile, Dashboard) is not displayed.

**Comment by hydegeoff** [comment]
Ticket can be closed
